Session.js

```diff
@@ -141,10 +144,29 @@ module.exports = class Session {
     return res;
   }
 
+  /**
+   * _startFreshIfExpired
+   * If the ttl indicates that the session has expired, treat this session object as if it were a new session.
+   */
+  _startFreshIfExpired() {
+    if (this.ttl <= 0) {
+      return;
+    }
+    const currentTime = Date.now();
+    if (currentTime <= this.updated + ttl) {
+      return;
+    }
+    // do whatever the constructor does
+    this.id = utils.randomString();
+    this.created = this.updated = currentTime;
```

Comment:
In JS, we typically don't do chain variable assignments. 
`this.created = currentTime; ` then `this.updated = currentTime;`

Reply:
I'll change that. I copied it from the constructor. I'll change that also

Reply:
done